ivre/passive.py

```diff
@@ -350,6 +360,24 @@ def _getinfos_dns(spec):
     return res
 
 
+def _getinfos_dns_blacklist(spec):
+    """Extract domain names in an handy-to-index-and-query form."""
```

Comment:
Nice try :). But could you describe what this function actually does? Something like `Extract and properly format DNSBL records.` should do.